.storybook/main.cjs

```diff
@@ -26,9 +26,8 @@ module.exports = {
   ],
   addons: [
     '@storybook/addon-a11y/register',
```

Comment:
Looks like docs, viewport and backgrounds addons could also be removed since they are apparantly part of the essentials

https://storybook.js.org/docs/react/essentials/introduction